RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -37,13 +37,13 @@ public struct CustomerCenterView: View {
     private var supportInformation: CustomerCenterConfigData.Support?
 
     /// Create a view to handle common customer support tasks
-    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil,
-                localization: CustomerCenterConfigData.Localization = .default,
-                appearance: CustomerCenterConfigData.Appearance = .default) {
+    /// - Parameters:
+    ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions from the customer center.
+    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil) {
```

Comment:
This is theoretically a breaking change... But I'm ok with it since this is still on beta only :)

Reply:
yes, if someone was using it, they shouldn't have lol